packages/astro/src/actions/runtime/virtual/client.ts

```diff
@@ -0,0 +1,18 @@
+export * from './shared.js';
+
+export function defineAction() {
+	throw new Error('[astro:action] `defineAction()` unexpectedly used on the client.');
+}
+
+export function getApiContext() {
+	throw new Error('[astro:action] `getApiContext()` unexpectedly used on the client.');
+}
+
+export const z = new Proxy(
```

Comment:
nit: can we have a better name instead of `z`?

Reply:
Wanted to stay consistent with content collections exposing `z`, since it _is_ the entirety of Zod. A name change would imply this is some superset of Zod.

Still, there's a general question here: **why can you import Zod from multiple modules in Astro?** It's now exposed from `astro:content` and `astro:actions`. Seemed out-of-scope to fix for the experimental release, but I want to discuss a change.

Reply:
I didn't know it was something around Zod, sorry about that